db/schema.rb

```diff
@@ -1278,11 +1280,11 @@
   add_foreign_key "active_storage_attachments", "active_storage_blobs", column: "blob_id"
   add_foreign_key "active_storage_variant_records", "active_storage_blobs", column: "blob_id"
   add_foreign_key "oauth_openid_requests", "oauth_access_grants", column: "access_grant_id", on_delete: :cascade
+  add_foreign_key "payment_intents", "stripe_records", column: "payment_record_id"
```

Comment:
This foreign key should not exist anymore. `payment_record_id` could well also point to the `paypal_records` table.
The foreign key made sense when we had only Stripe and there was only one table to point to, but now with the polymorphic approach you have to deliberately delete the previously existing foreign_key in your migration.

Reply:
Apparently this foreign key would _not_ be automatically created, but I can't find where it was specified in any migration file.

In any case, I've removed it via the migration.